### PR TITLE
Fix 67 simplify

### LINKED ISSUE #67: Simplify: skip py27, no conda-forge, pip pywps
## Description

Simplify the template:
* skip support for python 2.7
* add support for python 3.6, 3.7, ...
* conda-forge is troublesome for stabilitiy, avoid it.
* install pywps via pip

## Environment

* Cookiecutter version used, if any:
* Python version, if any:
* Operating System:

### PR DESCRIPTION
**Pull request by cehbrecht**
## Overview

This PR fixes #67.

Changes:

* Skip conda-forge
* Skip python 2.7 support
* Skip six
* Added Python 3.7 tests
* pep8
* update readme
* Added "restart" target to Makefile

## Related Issue / Discussion

Fix #67.

### CODE REVIEW

**Comment by cehbrecht** [comment]
@huard tested with babybird. It works.

File: environment.yml [hunk 1]
```diff
@@ -1,13 +1,18 @@
 name: cookiecutter-birdhouse
 channels:
-- conda-forge
+#- conda-forge
```
Comment (huard) [inline environment.yml]: I suggest we remove the comments to keep it clean.

Comment (cehbrecht) [reply environment.yml]: done.

**Comment by cehbrecht** [comment]
@huard done. Thanks for review :)

**Pull request merged**